### DISCUSSION TITLE
Error while running User Service [Ansible]

### DISCUSSION THREAD

**Discussion by Shravang007** [learndevopsonline/learndevopsonline#1046] [Q&A] [answered]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/133180652/a65b11db-cc4c-4380-92ec-de33d308d8c3)

Showing error like there is no user.service file but i have created that file. still it is throwing this error.

**Comment by learninguser** [comment]
Hi,

Its because that file is not present inside the `roles/common/files` directory. Kindly place the file in that directory and try to re-run your script once again. If the error still persits, please feel free to comment on this thread.

Thank you.
Kind regards
Pavan Kumar.

**Reply by Shravang007** [reply]
Thank you very much for your assistance. Problem resolved.

**Reply by learninguser** [reply, marked as answer]
As the issue is resolved, kindly mark this as answered.
